src/platforms/flutter/common/configuration/integrations/sqflite-instrumentation.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: sqflite Database Instrumentation
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.dart.sqflite
+description: "Learn more about the Sentry sqflite Database Instrumentation for the Flutter SDK."
+categories:
+  - mobile
+---
+
+<Include name="beta-note.mdx" />
+
+_(New in version 7.2.0)_
+
+The [sentry_sqflite](https://pub.dev/packages/sentry_sqflite) package provides `sqflite` support for Database Instrumentation. The source can be found [on GitHub](https://github.com/getsentry/sentry-dart/tree/main/sqflite).
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+## Install
+
+To add the sqflite Database Instrumentation, add the `sentry_sqflite` dependency.
+
+```yml {filename:pubspec.yaml}
+dependencies:
+  sentry: ^{{ packages.version('sentry.dart', '7.2.0') }}
+  sentry_sqflite: ^{{ packages.version('sentry.dart.sqflite', '7.2.0') }}
+  sqflite: ^2.0.0
+```
+
+## Configure
+
+There are different ways to configure the sqflite Database Instrumentation.
+
+Using the global `databaseFactory`, which is used by the `openDatabase` method, so every call to `openDatabase` will be instrumented, including from other packages:
+
+```dart
+import 'package:sentry_sqflite/sentry_sqflite.dart';
+import 'package:sqflite/sqflite.dart';
+
+databaseFactory = SentrySqfliteDatabaseFactory();
+final database = await openDatabase('path/to/db');
+```
+
+If you're using the `FFI` factories, you can use the `SentrySqfliteDatabaseFactory` with its factory:
+
+```dart
+import 'package:sentry_sqflite/sentry_sqflite.dart';
+import 'package:sqflite/sqflite.dart';
+
+databaseFactory = SentrySqfliteDatabaseFactory(databaseFactory: databaseFactoryFfi);
+final database = await openDatabase('path/to/db');
+```
+
+Using the `openDatabaseWithSentry` wrapper, only the database opened with this method will be instrumented:
+
+```dart
+import 'package:sentry_sqflite/sentry_sqflite.dart';
+import 'package:sqflite/sqflite.dart';
+
+final database = await openDatabaseWithSentry('path/to/db');
+// or final database = await openReadOnlyDatabaseWithSentry('path/to/db');
+```
+
+Using the `SentryDatabase` wrapper, which can be used to instrument any `Database` instance:
+
+```dart
+import 'package:sentry_sqflite/sentry_sqflite.dart';
+import 'package:sqflite/sqflite.dart';
+
+final database = await openDatabase('path/to/db');
+final sentryDatabase = SentryDatabase(database);
+```
+
+After doing this, every `CRUD` operation including transactions and batches will be automatically instrumented and sent to Sentry.
+
+The span's `operation`'s are either `db`, `db.sql.execute`, `db.sql.query` or `db.sql.transaction` and it's `description` is the SQL statement using placeholders instead of its values due to the possibility of containing PII.
```

Comment:
```suggestion
The spans' `operation`s are either `db`, `db.sql.execute`, `db.sql.query` or `db.sql.transaction`. Its `description` is the SQL statement using placeholders instead of its values due to the possibility of containing PII.
```